### DISCUSSION TITLE
樹懶消化也超慢？吃一餐消化一個月...

### DISCUSSION THREAD

**Discussion by Sherry9106** [1101-GS4719-C/Python-Community-GS4719-C#1585] [閒聊八卦]
樹懶的胃就占了體重的1/3，分成四個部分來慢慢消化食物，但樹懶吃的樹葉纖維強韌消化不易，因此樹懶吃了一餐後通常需要花上一整個月來消化食物。所以樹懶大概一個月吃一餐嗎?

**Comment by BOSENCHANG2004** [comment]
我常常在想如果我也是樹懶就好，生活步調慢，吃得也少省伙食費。我已經受夠了現代社會過快的生活步調了。
因為現代生活節奏太快了：
每天都在追進度、回訊息、完成任務
吃飯像打仗、走路像競賽、連放鬆都要排行程
有時候一整天下來，什麼都做了，但卻沒真的「活著」
這樣的狀態很容易讓人感到疲倦、空虛，
